src/server/knex.js

```diff
@@ -0,0 +1,73 @@
+// Define a Knex connection. Currently, this is used only to instantiate the
```

Comment:
Wondering if we should should rename this to `knex-config.js` for clarity. From the name I wouldn't have guessed that the whole file is a config

Reply:
maybe knex-connect.js?  the config is in environment variables, but this establishes the connection